docs/enable-apps.md

```diff
@@ -0,0 +1,138 @@
+# Enable the New Architecture for Apps
+
+This page will help you create or migrate a React Native app that uses the New Architecture.
+
+> [!WARNING]
+> If you're using Expo or plan to use Expo, you can't enable the New Architecture at the moment and will have to wait for a future release of the Expo SDK.
+
+### Prerequisites
+1. Use or upgrade to the latest React Native version. This guide is written with the expectation that you’re using the [**latest** React Native release](https://github.com/facebook/react-native/releases/latest).
+2. If you previously installed a global `react-native-cli` package, please remove it as it may cause unexpected issues:
+```shell
+  npm uninstall -g react-native-cli @react-native-community/cli
+```
+3. Enable Hermes. If you are using React Native 0.70 or above, it is already the [default JS engine](https://reactnative.dev/blog/2022/07/08/hermes-as-the-default) so no action is needed.
+
+### Create a React Native app
+
+If you are creating a new app to enable the New Architecture, follow all the steps in [Setting up the development environment](https://reactnative.dev/docs/environment-setup) section under the **React Native CLI Quickstart** tab.
+
+If following the setup guide, stop when you reach the section **Running your React Native Application**, and resume following this guide.
+
+Then, create a new React Native project from the template:
+
+```shell
+npx react-native@latest init AwesomeProject
```

Comment:
```suggestion
npx react-native@latest init AwesomeProject --skip-install
cd AwesomeProject
yarn install
```

Otherwise it will install yarn and Cocoapods automatically, and pods needs to be reinstalled anyway.